src/device-registry/bin/jobs/v3-store-readings-job.js

```
@@ -0,0 +1,267 @@
+    } else {
+      logDocumentDetails(doc);
+    }
+
+    // Wait for both updates to complete
+    await Promise.all(updatePromises);
+
+    // Update Reading
+    const filter = { site_id: doc.site_id, time: docTime.toDate() };
+    const updateDoc = { ...doc, time: docTime.toDate() };
+    delete updateDoc._id;
+    await ReadingModel("airqo").updateOne(filter, updateDoc, {
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)